src/java/us/kbase/templates/module_dockerfile.vm.properties

```diff
@@ -21,6 +23,11 @@ RUN pip install cffi --upgrade \
     && pip install requests --upgrade \
     && pip install 'requests[security]' --upgrade
 #end
+#if( $language == "java")
```

Comment:
We should only include this line if we are creating an example module.  There should be a flag available to indicate that it is an example.